tests/test-data/dir-nested.block

```diff
@@ -1,5 +1,4 @@
 5
-" $��G�,�A�4{���x�Z/.����D`�200Bytes.txt�3
```

Comment:
It was failing because git doesn't check in empty dirs. @nginnever has pushes a fix to his branch https://github.com/ipfs/js-ipfs-data-importing/pull/6/files which essentially creates the empty folders if they are not there..